MobilityData/mobility-feed-api
# feat: automate locations - modified API responses

**Summary:**

This PR updates the feed endpoints and the search endpoint to use the extracted locations and includes a new `country` field for each location within the `gtfs_feeds` response.

**Out of Scope:** 
- Filtering behaviour should use the english version of the locations in #662 (this is also needed to update the integration tests)

**Expected Behavior:**
- Each location in the response now includes a `country` field.
- All location fields are provided in English.

**Testing Tips:**
1. Access the UI link provided in the PR comments.
2. Navigate through the feeds.
3. Inspect the network tab response to verify the presence of the `country` field.
4. Ensure that all location fields are in English.

Please make sure these boxes are checked before submitting your pull request - thanks!
